### DISCUSSION TITLE
Équilibrage de stats des équipements

### DISCUSSION THREAD

**Discussion by Evatco** [Crownicles/Crownicles#2366] [Suggestions fermées]
On va pas se le cacher il y a des problèmes dans l'équilibrage des équipements d'une même classe ou des objets qui ne changent presque pas de stats en prenant un level. Bon je viens de voir des armes épiques sont mieux que des legs. Bref le but de ce post est de lister toutes les armes qui vont pas et qui ont besoin d'un réaussement de la stat de base ou d'en ajouter (ex items d'attaque + d'attaque ou donne un autre bonus)
Je me base sur le wiki donc ce qui y est pas tant pis
Les armes que j'ai remarqué nécessitant un équilibrage (si vous en avez, hésitez pas en coms) :
- Les armes légendaires d'attaque qui ont des stats inférieurs aux épiques et de grosses dif entre elles (Pistolet pan pan QQ, Bio-arme, Épée de Ragnell) 
- Armes mythiques qui ont bien besoin d'un boost surtout sabotage de l'oxygène :p
- Boucliers legs : Cri UwUrlant, Bouclier de maître En mythiques Blindage du major (une petite hausse globale de tous les mythiques ?)
- En objets : Nimbus 2000, Œil de Fatalis, Guitare terrifiante, Anneaux de défense (déjà que la def a plus d'importance 7 de plus va toout changer xD), Défibrillateur, Multinationale (15 de plus qu'un aimant XD), 
- En potions : Épinards
Voilà fini pour moi :p

**Comment by BastLast** [comment]
Hello c'est fait exprès

**Reply by Evatco** [reply]
Ah ok rip les armes leg moins bien que des épiques xD

**Reply by BastLast** [reply]
Plus rare != mieux

**Reply by Evatco** [reply]
Alors soit j'ai pas compris car il y a des objets épiques mieux que des legs (Pistolet chargé, épique 143 > 125 Pistolet pan pan QQ, legendaire)
Ou alors tu as lu depuis le bot discord où j'avais mal écrit mon com ^^
Où je comprends toujours rien xD

**Reply by BastLast** [reply]
> Alors soit j'ai pas compris car il y a des objets épiques mieux que des legs (Pistolet chargé, épique 143 > 125 Pistolet pan pan QQ, legendaire)
> Ou alors tu as lu depuis le bot discord où j'avais mal écrit mon com ^^
> Où je comprends toujours rien xD

J'ai tout bien lu

**Reply by Evatco** [reply]
D'accord c'était pour être sur ^^

**Comment by BananePlantain** [comment]
je suis d'accord que les objets d'énergie méritent un réel équilibrage qu'au mieux tu puisses regen entièrement avec le mythique donc 1800
